src/swapnaWJun20/PrimeOperations.java

```diff
@@ -0,0 +1,53 @@
+/*Program 1: Create a class to satisfy below requirements. 
+                  a) Given number is prime or not. 
+                  b) Print total count of prime numbers between 2->100.
+                  c) Find sum of all prime numbers between 2->100
+                  d) Find avg of prime numbers between 2->100*/
+
+package swapnaWJun20;
+
+public class PrimeOperations {
+
+	void totalCount(int a, int b) {
+		int sum = 0;
+		int count = 0;
+		for (int start = a; start < b; start++) {
+			boolean flag = false;
+			for (int index = 2; index <= Math.sqrt(start); index++) {
+				if (start % index == 0) {
```

Comment:
Why are you writing whole logic here, call isPrime() method.

Reply:
@KrishnaKTechnocredits  Changes done as per the comments.